research/Dockerfile

```diff
@@ -9,7 +9,8 @@ RUN apt-get update \
         libxml2-dev \
         libxslt-dev \
     && apt-get clean \
-    && rm -rf /var/lib/apt/lists/*
+    && rm -rf /var/lib/apt/lists/* \
+    && sudo -u jovyan mkdir /home/jovyan/.jupyter
```

Comment:
Would you mind adding `sudo -u jovyan mkdir /home/jovyan/.jupyter` as a single `RUN` step?

My point is that it doesn't have to do with the logic of the previous `RUN` (namely, with installing dependencies).

I think it makes sense to end up with a `RUN` step to install dependencies and another `RUN`step to fix permissions.

Does it make sense?